### ISSUE TITLE
How to use wireless pairing feature?

### ISSUE THREAD

**Issue by hiepnh0** [libimobiledevice/libimobiledevice#1274]
Running below command on Linux:
$ idevice_id -n
output is empty.

Any suggestions? Thank you so much.

**Comment by xanadu-lee** [comment]
Does the usbmuxd linux version implement wireless functionality?

**Comment by mexmer** [comment]
@xanadu-lee i don't think so ... there is fork that uses avahi for searching network with bonjour, fork is located here
https://github.com/tihmstar/usbmuxd2

**Comment by jkcoxson** [comment]
usbmuxd2 is not a fork, it's a complete rewrite in C++. I have also built an extension on top of vanilla usbmuxd for avahi.

**Comment by mexmer** [comment]
@jkcoxson i see, i didn't check code, so i did assume it's fork, because of name ... thanks for info.
